app/templates/frameworks/_submitted_services.html

```diff
@@ -13,8 +13,15 @@
       {% endif %}
     {% endfor %}
   </ul>
-  <a href="{{ url_for('.framework_submission_lots', framework_slug=framework.slug) }}">
-    View submitted services
-  </a>
+  <p>
+    <a href="{{ url_for('.framework_submission_lots', framework_slug=framework.slug) }}">
+      View submitted services
+    </a>
+  </p>
+  <p>
+    <a href="{{ url_for('.framework_supplier_declaration_overview', framework_slug=framework.slug) }}">
+      View your declaration
```

Comment:
There is a weird case if the supplier is marked on_framework but with an incomplete declaration - we'd show the link but it would 410 - but this should really really never happen. I can't think of any way we'd have a supplier on_framework without having completed their declaration - incomplete declaration means they didn't apply in the first place, so this should be fine I reckon.